docs-src/content/cli/Overview.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Overview"
+weight: 10
+---
+
+The cubic-cli binary provides a convenient way to generate transcripts from an audio file in order to evaluate a model.  It accepts either a single audio file or a list of files, sends the audio to a running instance of cubicsvr, and outputs the transcript in one of several formats.
+
+<!--more-->
+
+| Command                | Purpose |
+| ---------------------- | ------- |
+| `cubic-cli`            | Prints the default help message. |
+| `cubic-cli models`     | Displays the transcription models being served by the given instance. |
+| `cubic-cli version`    | Displays the versions of both the client and the server. |
+| `cubic-cli transcribe` | Sends audio file(s) to server for transcription. |
+
+`[cmd] --help` can be run on any command for more details on usage and included flags.
+
+## Global Flags
+
+All commands accept the following flags
+
+| Short | Long form      | Arg | Purpose |
+| ------|--------------- | ---- | ------- |
+| |      `--insecure`      | none | By default, connections to the server are encrypted (TLS).  Include`'--insecure` if you want TLS disabled. |
```

Comment:
I would put this below the `-s | --server` line.  Maybe this is a copy from the Readme, so I'm not overly worried.